### PR TITLE
chore(shard.yml): bump version for crystal 1.0.0

### PR DESCRIPTION
**Pull request by stakach**

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: future
-version: 0.1.0
+version: 1.0.0
 
 authors:
   - Brian J. Cardiff <bcardiff@manas.tech>
 
-crystal: 0.34.0
+crystal: ">= 0.34.0, < 2.0.0"
```
Comment (veelenga) [inline shard.yml]: why the `< 2.0.0` part is needed here?

Comment (stakach) [reply shard.yml]: in case of breaking changes in crystal 2.0.0?
happy to remove it

Comment (veelenga) [reply shard.yml]: I would remove it for now. Since there is no understanding if the library is going to be incompatible with 2.0

**Review by veelenga — APPROVED** [review]

**Comment by stakach** [comment]
@veelenga would you be able to tag a new version?
If you want I am happy to help manage this project?

**Comment by veelenga** [comment]
@stakach just invited you as a mainatiner. Let me know if you have any issues.

**Pull request merged**